### ISSUE TITLE
training loss nan

### ISSUE THREAD

**Issue by JiaquanYe** [ZhaoJ9014/face.evoLVe.PyTorch#42]
Thanks for sharing this fantastic repository!
But, when I train my own dataset, I have found training loss has become nan after few epoches.
Could you tell me how to solve this problem?

**Comment by ZhaoJ9014** [comment]
Hi, thank you for your question. Yes, sometimes due to different hard & soft environment "nan" may occur caused by training crash. You can try to decrease your learning rate and see:)

**Comment by JiaquanYe** [comment]
Thanks!




------------------ 原始邮件 ------------------
发件人: "Zhao Jian"<notifications@github.com>;
发送时间: 2019年4月24日(星期三) 下午3:30
收件人: "ZhaoJ9014/face.evoLVe.PyTorch"<face.evoLVe.PyTorch@noreply.github.com>;
抄送: "Villaaaa"<547885816@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [ZhaoJ9014/face.evoLVe.PyTorch] training loss nan (#42)




Hi, thank you for your question. Yes, sometimes due to different hard & soft environment "nan" may occur caused by training crash. You can try to decrease your learning rate and see:)
